package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "shared-chunks-plugin",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "format": "prettier --single-quote --trailing-comma es5 --tab-width 4 --write index.js",
```

Comment:
I run this manually before commit.  I can see about tying this in with a pre-push hook in a separate PR.

Reply:
Ok! This node module is super useful for that: https://github.com/observing/pre-commit Also, it might be worth moving the settings into a .prettierrc file so that you can share the settings with your IDE.